resources/views/api/master.blade.php

```diff
@@ -53,6 +43,18 @@
         <!-- footer -->
         <div>
             @include("api.includes.sections.footer")
-        </div>    
+        </div> 
+
+        <!-- <script src = "{!! load_asset('/js/api.js') !!}"></script>
+        <script src = "{!! load_asset('/js/api.js') !!}"></script>
+        <script src="{!! load_asset('/js/audio.min.js') !!}"></script>
+        <script src="{!! load_asset('/js/like.js') !!}"></script>
+        <script src = "{!! load_asset('/js/api.main.js') !!}"></script>
+        <script src="{!! load_asset('/js/audio.min.js') !!}"></script>
+        <script src = "{!! load_asset('/js/comment.js') !!}"></script> -->
```

Comment:
**Code Quality - Nitpick**

Any reason why we have all these scripts comment that do absolutely nothing @andela-araimi ?